#request
int8 index
bool is_enabled
float32 speed
bool direction
---
#response
bool success
